tests/cli_integration.rs

```diff
@@ -238,7 +239,7 @@ mod cli_integration {
 	}
 
 	#[test(tokio::test)]
-	async fn start_tls() {
+	async fn start_tls() -> () {
```

Comment:
This is neither necessary nor idiomatic. Not returning anything implicitly returns a unit already.